alf/algorithms/agent.py

```diff
@@ -100,12 +105,22 @@ def __init__(self,
             """
         agent_helper = AgentHelper(AgentState)
 
-        ## 1. goal generator
+        # 0. representation learner
```

Comment:
A minor point: the other items used ## (e.g. ## 1, ## 2). Here we used # 0. Maybe better to make them consistent?